frontend/javascripts/router.js

```diff
@@ -488,7 +488,22 @@ class ReactRouter extends React.Component<Props> {
                         Enum.coalesce(TracingTypeEnum, match.params.type) ||
                         TracingTypeEnum.skeleton;
                       const withFallback = match.params.withFallback === "true";
-                      const annotation = await createExplorational(dataset, type, withFallback);
+
+                      const params = Utils.getUrlParamsObjectFromString(location.search);
+
+                      const resolutionRestrictions = {};
+                      if (params.minRes !== undefined) {
+                        resolutionRestrictions.min = parseInt(params.minRes);
+                      }
+                      if (params.maxRes !== undefined) {
+                        resolutionRestrictions.max = parseInt(params.maxRes);
```

Comment:
Looks like the frontend sends null with the current code if something that is not parseable as int is passed. The backend interprets this as unlimited maxRes. Do you think this is acceptable?

Reply:
Hm, throwing an error here would be better in my opinion. Otherwise, potential errors (e.g., when constructing the URL) are swallowed silently. You can use something like `if (!_.isNumber(resolutionRestrictions.min)) { throw new Error("Invalid minRes parameter") } .